Issue #283 - Add new tex glossary and remove old files

mdipirro opened the pull request:
_Numero del Task/Issue correlato/a_: #283 

_Breve descrizione della soluzione adottata_:
Add new tex glossary and remove old files. Now is possible to compile the glossary using the standard command.

Polpetta committed:
Rename RP to RQ in Makefile

See isuee #282

mdipirro committed:
Add new tex glossary and remove old files

mdipirro committed:
Minor change

mdipirro committed:
Add some files previously deleted

mdipirro commented:
@Polpetta questo è il glossario che dovrebbe compilare bene con il comando classico. Nei test parte quello vecchio quindi non passano, ma da me compila, parola di boy scout!!

mdipirro committed:
Add xml glossary file

mdipirro commented:
Ho aggiunto anche il file xml. L'xml esportato da Tracy, oltre a non essere conforme allo script (cosa superabile) non riporta nemmeno i plurali o la descrizione estesa degli acronimi, quindi per me è da scartare

Polpetta commented:
@mdipirro lo so, stavo lavorando sulla mia issue per aggiustare questa cosa. Vuoi che mergio il mio task sul tuo? In questa maniera passerebbe i test e sarebbe mergiabile. Fammi sapere!

mdipirro commented:
fai pure

Polpetta commented:
Ok. Ora son a lezione quindi non faccio, appena finito in treno mi metto!
Ti faccio una pull request del mio task su questo task e metto te come
verificatore, così puoi controllare quello che ho fatto e "ci copriamo le
spalle" a vicenda!

Il mio task e il tuo fanno parte della stessa attività?:)
Il 28 apr 2016 16:54, "Matteo Di Pirro" notifications@github.com ha
scritto:

fai pure

—
You are receiving this because you were mentioned.

Reply to this email directly or view it on GitHub
https://github.com/BugBusterSWE/documentation/pull/367#issuecomment-215452254

mdipirro commented:
si fanno parte della stessa attività

Polpetta commented:
Ottimo!
Il 28 apr 2016 17:06, "Matteo Di Pirro" notifications@github.com ha
scritto:

> si fanno parte della stessa attività
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/BugBusterSWE/documentation/pull/367#issuecomment-215456017

Polpetta committed:
Add xml-glossary submodule

This perl script can generate a glossary for a xml.
See issue #282

Polpetta committed:
Add new function for build glossary from a xml

This function call the xml-glossay perl script and build
the glossary.tex file.
See issue #282

Polpetta committed:
Add perl in the travis build

See issue #282

Polpetta committed:
Minor changes to build system

See issue #282

Polpetta committed:
Merge branch 'issue#283' into issue#282

Add xml in this branch, so we can test the new Makefile

Polpetta committed:
Add new file and modify build.sh

See issue #282

Polpetta committed:
Update with a new version of xml-glossary

Polpetta committed:
Add comment for debug

Polpetta committed:
Add double brackets

Polpetta committed:
Removed glossary.tex and added to .gitignore files

Polpetta committed:
Rever changes

Polpetta committed:
Another testing condition

Polpetta committed:
Minor fix

Polpetta committed:
Update xml-glossary with new fix

mdipirro committed:
Merge pull request #375 from BugBusterSWE/issue#282

Issue#282 - Add support for xml
